spec/features/audit/navigation_spec.rb

```diff
@@ -71,23 +69,4 @@
     expected = content_item_audit_path(second, some_filter: "value")
     expect(current_url).to end_with(expected)
   end
-
```

Comment:
Could you please add a test for this feature? As we are not able to precalculate the link, shouldn't we test that we see the expected message once we try to navigate to the latest item?